fenix/app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleUtils.kt

```diff
@@ -0,0 +1,232 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.advanced
+
+import java.util.Locale
+
+/**
+ * Helper class for displaying Locale name.
+ */
+object LocaleUtils {
+
+    /**
+     * Returns a name for the locale that is appropriate for display to the user.
+     */
+    fun getDisplayName(locale: Locale): String {
+        val displayName = locale.getDisplayName(locale)
```

Comment:
Noticed this seems different than the original.

Original was: 
`.capitalize(locale)`

This has:
```
.replaceFirstChar { if (it.isLowerCase()) it.titlecase(locale) else it.toString() }
```

Maybe expose `private fun String.capitalize(locale: Locale)`,  [here](https://searchfox.org/mozilla-mobile/rev/32c38f958200af417e14887c4e0003f434f9c15f/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleViewHolders.kt#323) or move it to this utility file?

Reply:
val displayName = locale.getDisplayName(locale).capitalize(locale)
"capitalize" method is deprecated.